pkg/api/v1/types.go

```diff
@@ -1241,6 +1243,10 @@ type PodSpec struct {
 	ImagePullSecrets []LocalObjectReference `json:"imagePullSecrets,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
 }
 
+// PodSecurityContext holds pod-level security attributes and common container settings.
+type PodSecurityContext struct {
```

Comment:
Let's not add this until it has something in it. The 1.1 deadline is Monday. An empty field will just be confusing.